recipes/libcurl/all/conanfile.py

```diff
@@ -1,13 +1,21 @@
-import glob
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.apple import is_apple_os, fix_apple_shared_install_name
+from conan.tools.build import cross_building
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.env import VirtualBuildEnv
+from conan.tools.files import apply_conandata_patches, copy, download, get, load, replace_in_file, rm, rmdir, save
+from conan.tools.gnu import Autotools, AutotoolsToolchain, AutotoolsDeps, PkgConfigDeps
+from conan.tools.microsoft import is_msvc
+from conan.tools.microsoft.subsystems import unix_path
```

Comment:
```suggestion
from conan.tools.microsoft import is_msvc, unix_path
```

Never import conan public api with from conan.xxx.yyy.zzz